package.json

```diff
@@ -10,8 +10,7 @@
     "prettier-check": "prettier --check **/*.{js,jsx,scss,css,html}",
     "prettier-format": "prettier --write **/*.{js,jsx,scss,css,html}",
     "test": "yarn prettier-check && yarn workspace cli test && yarn workspace client test",
-    "build": "yarn workspace client build && yarn workspace cli start && cd stumptown && npm run build-json html",
-    "build-content": "cd stumptown && npm run build-json html && cd .. && yarn workspace cli start",
+    "build": "cd stumptown && npm run build-json html && cd .. && yarn workspace client build && yarn workspace cli start",
```

Comment:
The diff might be hard to read but what I did was I deleted the `build-content` line and in the `build` line I made it so that the stumptown stuff runs first.